repo-agent/repowatch/controllers/repowatch_controller.go

```diff
@@ -917,8 +917,9 @@ func (r *RepoWatchReconciler) createReviewSandboxForPR(ctx context.Context, repo
 					"name": repoWatch.Spec.Review.LLM.Provider,
 				},
 				"llm": map[string]interface{}{
-					"configdirRef": repoWatch.Spec.Review.LLM.ConfigdirRef,
-					"prompt":       prompt,
+					"configdirRef":     repoWatch.Spec.Review.LLM.ConfigdirRef,
+					"prompt":           prompt,
```

Comment:
The key `apiKeySecretName` is used here, but in the RGD YAML files, it's `apiKeySecretName`. This is consistent, which is good. However, the `RepoWatch` API uses `APIKeySecretRef`. For consistency, consider renaming `apiKeySecretName` to `apiKeySecretRef` in the sandbox RGD and the controller.